hangman.py

```diff
@@ -1,76 +1,74 @@
-'''
+"""
 The hangman game is a word guessing game where the player is given a word and has to guess the letters that make up the word. 
 The player is given a certain number of tries (no more than 6 wrong guesses are allowed) to guess the correct letters before the game is over.
-'''
+"""
 
 # Output
-'''
+"""
 You have 6 tries left.                                                                                                                                           
 Used letters:                                                                                                                                                    
 Word: _ _ _ _                                                                                                                                                    
 Guess a letter: a 
-
 You have 6 tries left.                                                                                                                                           
 Used letters: a                                                                                                                                                  
 Word: _ a _ a                                                                                                                                                    
 Guess a letter: j    
-
 You have 6 tries left.                                                                                                                                           
 Used letters: j a                                                                                                                                                
 Word: j a _ a                                                                                                                                                    
 Guess a letter: v                                                                                                                                                
 You guessed the word java !
-'''
+"""
 
-count = 0
-count_balance = 6
+stored_letters = list("java")  # converts string to list

```

Comment:
This is the same as doing `["java"]` I'm not sure what you mean by *converts* but to be clear the result is not `["j", "a", "v", "a"]`